openprescribing/frontend/price_per_unit/savings.py

```diff
@@ -0,0 +1,144 @@
+import numpy
+
+from matrixstore.db import get_db, get_row_grouper
+from matrixstore.matrix_ops import get_submatrix
+from matrixstore.sql_functions import MatrixSum
+
+from .substitution_sets import get_substitution_sets
+
+
+# Defines how we determine the target PPU against which savings are calculated.
+# We want to know which set of practices we are comparing with and which
+# centile over those practices we are targeting.
+CONFIG_TARGET_CENTILE = 10
+# Note this is "standard_practice" rather than "practice" as we only want to
+# include setting 4 practices (i.e. ordinary GP practices)
+CONFIG_TARGET_PEER_GROUP = "standard_practice"
+
+# Savings are calculated in two stages: first at the practice level, and then
+# aggregated up into organisation savings. At both stages we apply a minimum
+# threshold such that savings below that threshold are treated as zero. This is
+# particularly important at practice level as it prevents "negative savings"
+# from practices already performing better than the target PPU.
+# (Note: all values in pence)
```

Comment:
From this comment alone, it's hard to understand what is meant here.

The rationale for treating small savings as zero is (IIRC) that trivial savings are unlikely ever to be worth making.

The negative savings thing is handled separately and not really relevant to this config variable, I think?

It would also be useful to be clear what the `all_standard_practices` option means, i.e. savings under this at all-England level are considered trivial (right?)